lib/transformers/cloudinary.js

```diff
@@ -50,11 +50,42 @@ function buildCloudinaryTransforms(transform) {
 		cloudinaryTransforms.effect = `tint:100:${tint.join(':')}`;
 	}
 
+
 	const immutable = transform.getImmutable();
 	if (immutable) {
 		cloudinaryTransforms.flags.push('immutable_cache');
 	}
-	return cloudinaryTransforms;
+
+	const overlay = transform.getOverlay();
+
+
+	if (overlay) {
+
+		const overlayPosition = transform.getOverlayPosition();
+		const overlayHeight = transform.getOverlayWidth();
+		const overlayWidth = transform.getOverlayHeight();
```

Comment:
Oops! Even when I found the problem, it took me so long to see that this was the cause 🙈 

```suggestion
		const overlayHeight = transform.getOverlayHeight();
		const overlayWidth = transform.getOverlayWidth();
```

Reply:
Well spotted!